charts/tezos-proto-cruncher/scripts/proto-downloader.py

```
@@ -0,0 +1,17 @@
+import os
+
+BUCKET_NAME = os.getenv("BUCKET_NAME")
+BUCKET_ENDPOINT_URL = os.getenv("BUCKET_ENDPOINT_URL")
+BUCKET_REGION= os.getenv("BUCKET_REGION")
+
+PROTO_NAME = os.getenv("PROTO_NAME")
```

Maybe fail fast via another getter if the env vars don't exist? Say endpoint doesn't exist. The url will be created with `None`. Might as well fail fast and the stack trace will clearly show why.